pages/index.css

```diff
@@ -6,6 +6,31 @@
 @import url(../blocks/page/page.css);
 @import url(../blocks/page/__container/page__container.css);
 
+/* Price Block */
+@import url(../blocks/price/price.css);
+@import url(../blocks/price/__toggles/price__toggles.css);
+@import url(../blocks/price/__toggles/-slider/price__toggles-slider.css);
+@import url(../blocks/price/__toggles/-slider/_current/price__toggles-slider_current.css.css);
+
+
+
+/*Table Block */
+@import url(../blocks/table/table.css);
+@import url(../blocks/table/__row/table__row.css);
+@import url(../blocks/table/__header/table__header.css);
+@import url(../blocks/table/__header/_hit/table__header_hit.css);
+@import url(../blocks/table/__plan-name/table__plan-name.css);
+@import url(../blocks/table/__plan-price/table__plan-price.css);
+@import url(../blocks/table/__cell/table__cell.css);
+@import url(../blocks/table/__point/table__point.css);
+@import url(../blocks/table/__cell/-content/table__cell-content.css);
+@import url(../blocks/table/__cell/_content/table__cell_content_name.css);
```

Comment:
`table__cell_content_name` is a modifier of the `table__cell` element. Please, import it below `table__cell` element. But, please, have a read of the comment on the naming of this modifier above.

Reply:
@5hraddha I did it